app/server/lib/ActiveDoc.ts

```diff
@@ -2135,6 +2135,9 @@ export class ActiveDoc extends EventEmitter implements AssistanceDoc {
 
   private async _checkDataSizeLimitRatio(docSession: OptDocSession | null) {
     const start = Date.now();
+    if (!this.docStorage.isInitialized()) {
+      return;
+    }
```

Comment:
These line are not directly related to the issue of this PR. They exist in order to fix this exception:
```
2023-07-12 11:14:39.717 - error: ActiveDoc failed to update data size Error: Tried to use DocStorage database before it was opened
```